mitodl/micromasters
# Add fixed price coupons

#### What are the relevant tickets?
Fixes #2399 

#### What's this PR do?
Adds support for fixed price coupons

#### How should this be manually tested?
Create a fixed price coupon, attach your user to the coupon, then verify that the price you see in the UI is the one you set for the coupon

Note: the order summary messaging hasn't changed. For a $50 fixed price coupon on a $123 price it will look like this:
![screenshot from 2017-02-01 17-09-34](https://cloud.githubusercontent.com/assets/863262/22528479/7318d284-e8a1-11e6-8e48-d559aa7e9cc1.png)
